### PR TITLE
Loosen fuzzy query config

### PR DESCRIPTION
**Pull request by orangejenny**
## Technical Summary
https://dimagi-dev.atlassian.net/browse/USH-2095

This will make fuzzy queries more permissive, returning more results.

[This question](https://stackoverflow.com/questions/50368459/why-is-my-elastic-search-prefix-query-case-sensitive-despite-using-lowercase-fil) recommends lowercasing the search query, since we use a lowercase filter: https://github.com/dimagi/commcare-hq/blob/7124556769698e87f93314616bc9632b1e63e795/corehq/ex-submodules/pillowtop/es_utils.py#L20

The exact behavior of `max_expansions` is somewhat nebulous. From [this commentary](https://medium.com/@mourjo_sen/a-detailed-comparison-between-autocompletion-strategies-in-elasticsearch-66cb9e9c62c4):

> Note that this limit on expansions is done per-shard, do depending on the data on the shard, your results will vary, which makes this quite non-deterministic from the client point-of-view.

`max_expansions` defaults to 50 (see docs [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-fuzzy-query.html)), so I'm doubling it.

## Feature Flag
Case search

### Safety story

The major risk here is performance, as noted in the [ES docs on max_expansions](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-fuzzy-query.html). I've done some testing in the reported domain and our largest CICT domain (results in ticket).

### Automated test coverage

In PR.

### QA Plan

Not requesting QA.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/es/queries.py [hunk 1]
```diff
@@ -111,6 +113,18 @@ def match(search_string, field, fuzziness="AUTO", operator=None):
     }
 
 
+def fuzzy(search_string, field, fuzziness="AUTO"):
+    return {
+        "fuzzy": {
```
Comment (esoergel) [inline corehq/apps/es/queries.py:116]: Did this end up behaving differently than a `match` query with `fuzziness` set?  I don't recall.

**Review by esoergel — APPROVED** [review]
Any idea whether we should be concerned about how this change will affect users?  I think it's likely to be better behavior, but it is a change, so say, any saved searches might now return different results.

File: corehq/apps/es/queries.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/es/queries.py:116]: I think they behave the same except that the `match` query doesn't allow modifying `max_expansions`.

**Comment by orangejenny** [comment]
@esoergel Is there anywhere we support saved searches with fuzziness? Web apps doesn't let you save anything. Maybe a CLE query using the new fuzzy search function? But that'd be USH-only. I'm wary of making an announcement anyway, I think it'd be too vague to be useful.

**Pull request merged**